src/processors/file-handler.ts

```diff
@@ -47,25 +50,25 @@ const JSON_LOADER_OPTIONS = {
   ]
 };
 
-export function isGeoJson(json) {
+export function isGeoJson(json: any): boolean {
   // json can be feature collection
   // or single feature
   return isPlainObject(json) && (isFeature(json) || isFeatureCollection(json));
```

Comment:
add type `isPlainObject(obj: unknown): obj is Record<string, unknown>`

Reply:
added